crds/bestrefs/bestrefs.py

```diff
@@ -12,6 +12,7 @@
 # ===================================================================
 
 import crds
+import logging
```

Comment:
CRDS has a specific log module. Is there a reason for not using that?

Reply:
No not necessarily. This import isn't actually used at all and I think was just an artifact of some testing attempts. I'll comment out unused imports in the next commit.